config/v1/types_oauth.go

```diff
@@ -40,8 +40,27 @@ type OAuthSpec struct {
 	// templates allow you to customize pages like the login page.
 	// +optional
 	Templates OAuthTemplates `json:"templates"`
+	// Audit specifies what needs to be audited.
+	// + optional
+	Audit OAuthAudit `json:"audit"`
 }
 
+type OAuthAudit struct {
+	// Simple drop in profile type.
+	Profile OAuthAuditProfileType `json:"profile,omitempty"`
```

Comment:
We would need to set here a default value here with the kubebuilder, right?

https://github.com/openshift/enhancements/pull/894/files#r721093860

Reply:
yes